packages/client/src/api/api.service.ts

```diff
@@ -70,6 +70,14 @@ export class ApiService {
     return await this.httpClient.delete(`/widgets/messages/${messageId}`, {});
   }
 
+  async removeAllMessages(feedId?: string): Promise<any> {
+    const url = feedId
+      ? `/widgets/messages?feedId=${feedId}`
+      : `/widgets/messages`;
```

Comment:
This is dangerous, to allow to delete all messages if a feedId is missed for any reason.

Reply:
feedId is optional, user can remove either feed specific message or all messages